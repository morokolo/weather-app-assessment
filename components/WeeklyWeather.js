import React from "react";
import { StyleSheet, Text, View } from "react-native";

export default function WeeklyWeather() {
  return (
    <View>
      <Text>WeeklyWeather</Text>
    </View>
  );
}

const styles = StyleSheet.create({});
