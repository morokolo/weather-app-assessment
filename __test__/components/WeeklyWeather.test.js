import "react-native";
import React from "react";
import WeeklyWeather from "../../components/WeeklyWeather";
import renderer from "react-test-renderer";

import { configure } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

configure({ adapter: new Adapter() });

it("should render the WeeklyWeather functional component", () => {
  const tree = renderer.create(<WeeklyWeather />).toJSON();
  expect(tree).toMatchSnapshot();
});
