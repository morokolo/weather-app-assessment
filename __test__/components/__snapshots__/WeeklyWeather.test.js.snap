// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render the WeeklyWeather component with  send props 1`] = `
<View>
  <View
    style={
      Array [
        Object {
          "flexBasis": 60,
          "flexDirection": "row",
          "padding": 15,
          "paddingBottom": 0,
        },
        Object {
          "borderBottomColor": "#fff",
          "borderBottomWidth": 1,
          "flexBasis": 70,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#fff",
          }
        }
      >
        10.05
        째
      </Text>
      <Text
        style={
          Object {
            "color": "#fff",
          }
        }
      >
        min
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#fff",
            },
            Object {
              "alignSelf": "center",
            },
          ]
        }
      >
        12.58
        째
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#fff",
            },
            Object {
              "alignSelf": "center",
            },
          ]
        }
      >
        Current
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "color": "#fff",
            },
            Object {
              "alignSelf": "center",
            },
          ]
        }
      >
        16.96
        째
      </Text>
      <Text
        style={
          Array [
            Object {
              "color": "#fff",
            },
            Object {
              "alignSelf": "center",
            },
          ]
        }
      >
        max
      </Text>
    </View>
  </View>
  <View
    style={
      Array [
        Object {
          "flexBasis": 60,
          "flexDirection": "row",
          "padding": 15,
          "paddingBottom": 0,
        },
        Object {
          "backgroundColor": undefined,
        },
      ]
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Text
        style={
          Object {
            "color": "#fff",
          }
        }
      >
        Monday
      </Text>
    </View>
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Image
        source={1}
        style={
          Object {
            "alignSelf": "center",
            "height": 30,
            "width": 30,
          }
        }
      />
    </View>
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <Text
        style={
          Array [
            Object {
              "alignSelf": "center",
              "fontSize": 20,
              "marginLeft": 12,
            },
            Object {
              "color": "#fff",
            },
          ]
        }
      >
        17
        째
      </Text>
    </View>
  </View>
</View>
`;
